handle.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"net/url"
+	"strconv"
+	"time"
+)
+
+type paramkey string
+
+const (
+	paramkeyDelay paramkey = "delay"
+	paramkeyFib   paramkey = "fib"
+)
+
+func handle(w http.ResponseWriter, r *http.Request) {
+	params := r.URL.Query()
+
+	delayMS, err := readParamInt(params, paramkeyDelay)
+	if err != nil {
+		respondError(w, 400, err)
+		return
+	}
+
+	fibIndex, err := readParamInt(params, paramkeyFib)
+	if err != nil {
+		respondError(w, 400, err)
+		return
+	}
+
+	if delayMS > 0 {
+		time.Sleep(time.Duration(delayMS) * time.Millisecond)
+	}
+
+	if fibIndex > 0 {
+		fibonacci(fibIndex) //nolint
+	}
```

Comment:
It does not respond anything right ?
No body written could be ok (is it?) but not even a status ? 200 is the default ?

This seems kinda strange to not even declare what we're sending back to me.

Reply:
It's a 200 with empty body by default, so to me it doesn't seem necessary to explicit it